### DISCUSSION TITLE
login or session export

### DISCUSSION THREAD

**Discussion by EverythingGoesWithTheFlow** [kingToolbox/WindTerm#69] [Ideas]
Would you consider adding user login or session export so that it can be used in multiple places

**Comment by kingToolbox** [comment]
Would you like to describe in detail the content and format you want to export? And the detailed configuration of each session is saved in the file `windterm-install-path/profiles/default.v10/terminal/user.sessions`, in json format. I think it should be very easy for third parties to read and write.

Of course, `session.autoLogin` is encrypted and cannot be exported. Otherwise, there is no safety at all.

**Reply by EverythingGoesWithTheFlow** [reply]
thank you

**Reply by kingToolbox** [reply]
One more thing, the design of WindTerm with open format configuration is to facilitate users to import and export, even if users want to migrate to other clients, they can also migrate very easily. :smile:
